app/src/main/cpp/VRVideo.cpp

```diff
@@ -260,6 +260,23 @@ struct VRVideo::State {
     leftEye = create180LayerToggle(equirect);
   }
 
+#ifdef OPENXR
+  void create180LRProjectionLayer() {
+    vrb::CreationContextPtr create = context.lock();
+    DeviceDelegatePtr device = deviceWeak.lock();
+    VRLayerEquirectPtr equirect = device->CreateLayerEquirect(window->GetLayer());
+    layer = equirect;
+
+    equirect->SetTextureRect(device::Eye::Left, device::EyeRect(0.0f, 0.0f, 0.5f, 1.0f));
+    equirect->SetTextureRect(device::Eye::Right, device::EyeRect(0.5f, 0.0f, 0.5f, 1.0f));
+    auto UVtransform = vrb::Matrix::Identity().Scale(vrb::Vector(2.0f, 1.0f, 1.0f)).Translate(vrb::Vector(-0.5, 0.0, 0.0));
+    equirect->SetUVTransform(device::Eye::Left, UVtransform);
+    equirect->SetUVTransform(device::Eye::Right, UVtransform);
+    equirect->SetUseSameLayerForBothEyes(false);
```

Comment:
That's the advantage of OpenXR, we can show different images to each eye using a single layer, with the OculusVR backend you had to create 2 layers.

Reply:
> I understand the confusion, the problem I think here is naming. It's indeed using the same XR layer, but it's using different portions of that layer which are set with the `SetTextureRect` above. I added that method when fixing 3D side by side to avoid submitting the same image to both eyes. Maybe I could rename it to something much better in a follow up. WDYT?

Ok, I see. It is indeed confusing. I'm fine with clarifying this in a future patch, I expect we won't have to touch this code often anyways.